src/components/compound/DlCharts/charts/DlConfusionMatrix/DlConfusionMatrix.vue

```diff
@@ -546,6 +546,10 @@ export default defineComponent({
     transform: translateX(40%);
     display: block;
     text-align: start;
+    width: 60px;
```

Comment:
This will simply hide the overflow. What is the desired behavior though?

Reply:
The desired behavior is to hide the overflowing content by showing ellipsis and by showing the full string on hover.